RevPiDevice.c

```diff
@@ -243,7 +243,7 @@ int RevPiDevice_run(void)
 
 	// if the user-ioctl want to send a telegram, do it now
 	if (test_bit(REVPI_CORE_FLAG_PENDING_TELEGRAM, &piCore_g.flags)) {
-		piCore_g.statusUserTel = piIoComm_sendTelegram(&piCore_g.requestUserTel, &piCore_g.responseUserTel);
+		piCore_g.statusUserTel = piIoComm_sendTelegram(&piCore_g.requestUserTel);
```

Comment:
Since `piIoComm_sendTelegram()` is essentially just turned into a wrapper for `pibridge_req_io()` by this commit and this is its only invocation in the entire code base, why not call `pibridge_req_io()` directly here and delete `piIoComm_sendTelegram()` altogether?

Reply:
Mainly for consistency. We do not call any pibridge API() directly from within RevpiDevice.c until now. But I am also fine with removing the piIoComm_sendTelegram() wrapper.